assets/app/features/upgrade/templates/landing-page.jade

```diff
@@ -11,3 +11,4 @@
     .row.hidden-md.hidden-lg.step-3(ng-class="{'enabled': upgradeLandingVm.mode.valid}")
         .col-md-6
             include landing-page_step3.partial.jade
+suse-modal(error="upgradeLandingVm.error")
```

Comment:
I think the controller should be the one creating/deleting modals as error appears.